### DISCUSSION TITLE
[Question] When will the server be recovered after planned power outage?

### DISCUSSION THREAD

**Discussion by retroinspect** [prosyslab-classroom/cs424-program-reasoning#339] [Q&A]
There is planned outage on 11/6 Sunday, from 9 AM to 6PM.
I wonder when the server for our projects to be recovered after then.

**Comment by sujin0529** [comment]
The network will be disconnected from 8 a.m. to 7 p.m. So I think the server will be recovered after 7 p.m.
Sorry, I don't know exactly.
